code/game/dna/dna2.dm

```diff
@@ -52,7 +52,7 @@
 #define DNA_UI_TAIL_MARK_STYLE	32
 #define DNA_UI_LENGTH		32 // Update this when you add something, or you WILL break shit.
 
-#define DNA_SE_LENGTH 55 // Was STRUCDNASIZE, size 27. 15 new blocks added = 42, plus room to grow.
+#define DNA_SE_LENGTH 56 // Was STRUCDNASIZE, size 27. 15 new blocks added = 42, plus room to grow.
```

Comment:
Hm, are you sure this value needs to be increased? I thought we still had empty blocks left?